clang/include/clang/Basic/Attr.td

```diff
@@ -1239,6 +1239,23 @@ def SYCLSimdAccessorPtr : InheritableAttr {
   let Documentation = [Undocumented];
 }
 
+// The attribute denotes that it is a function written in SIMT fashion, which
+// is used in ESIMD context and needs to be widened by a vector backend compiler.
+// For now, this attribute will be used only in internal implementation of
+// specific public ESIMD APIs. It is not supposed to be used directly in the
+// user code, hence it is undocumented.
+// The argument of the attribute specifies the number of SIMD lanes, for which
+// the function should be widened.
+def SYCLIntelESimdWiden : InheritableAttr {
+  let Spellings = [GNU<"sycl_esimd_widen">, CXX11<"intel", "sycl_esimd_widen">];
```

Comment:
"widening" is kind of non-standard term, "vectorization" is. I'd suggest something like "sycl_esimd_auto_vectorize" (other variants - "sycl_esimd_vectorize", "sycl_esimd_vector").

Reply:
Ok, I talked to a member of the vectorization team and they suggested that "vectorization" would be indeed better. I will go with `sycl_esimd_vectorize`.

Reply:
ok